src/Storefront/Theme/StyleCompilerInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Storefront\Theme;
+
+use Shopware\Storefront\Theme\StorefrontPluginConfiguration\StorefrontPluginConfiguration;
+
+interface StyleCompilerInterface
+{
+    public function compileStyles(string $variables, string $concatenatedStyles, StorefrontPluginConfiguration $configuration, array $resolveMappings, string $salesChannelId): string;
```

Comment:
Just as an Idea passing only a Struct `CompileContext` instead so many arguments. So in future we don't need a major to add a new argument :)

What do you think about that?

Reply:
I like the idea. Who needs to decide on that?

Reply:
just do it 😁